src/chain.h

```diff
@@ -403,6 +407,7 @@ class CDiskBlockIndex : public CBlockIndex
         READWRITE(hashPrev);
         READWRITE(hashMerkleRoot);
         READWRITE(nTime);
+        READWRITE(block_height);
```

Comment:
I don't understand why this doesn't need to be conditional. Bitcoin block indexes sure won't have height here...

Reply:
I'm not sure myself, but Greg discussed it with Jorge here: https://github.com/ElementsProject/elements/pull/431#discussion_r225684601